.github/tests/holesky-shadowfork-verkle.yaml_norun

```diff
@@ -10,7 +10,7 @@ participants:
 network_params:
   electra_fork_epoch: 1
   network: holesky-shadowfork-verkle
-  genesis_delay: 300
+  #network_sync_base_url: localhost:10000/snapshots/
```

Comment:
why keep a comment?

Reply:
its an example, to make it easy to configure

Reply:
its not ran anyway by anyone but those that run shadowforks daily